packages/secret-phrase-pcd/dist/index.js

```diff
@@ -0,0 +1,1205 @@
+"use strict";
```

Comment:
we should probably `.gitignore` this file?

Reply:
Oops thought everything would be picked up by top level gitignore. fixed